README.md

```diff
@@ -128,3 +128,26 @@ Thanks and keep hacking!
 Cheers,  
   -Brian  
   Seattle, WA.
+
+Releasing Gearmand
+------------------
+
+When it's time to release the master branch, check the last release tag, and
+create a new one. Most releases are patch releases, and so should increment the
+third number in the version.
+
+- Checkout master
+- Ensure there are no changes staged or un-added. Anything not in git master will be removed by the next step!
+- ```$ git clean -xdf # This will remove all generated files```
+- Review changes merged since the last release with git log $LAST_RELEASE_TAG..
+- ```$ git tag -s $NEW_RELEASE_TAG # Type in release notes```
+- ```$ ./bootstrap.sh -a && ./configure # This is needed to make dist properly```
+- ```$ make dist # This creates the tarball```
```

Comment:
I found that I need to execute `make -C docs man` before `make dist` works?

Reply:
Part of the reason I did this as a branch instead of a fork pull is that I wanted you to be able to push. If you want to go ahead and push a fix to the branch with the docs command, I think you're more likely to get it right than I am.

Thanks Ed!

Reply:
None of the instructions I've tried for pushing commits to someone else's pull request seem to work. So I'm just going to merge this and put my fixes in a separate PR.